### DISCUSSION TITLE
Kubernetes Labels not in Resultfile

### DISCUSSION THREAD

**Discussion by mirisbowring** [prowler-cloud/prowler#5733] [Q&A]
Hi,
I was investigating the structure of the prowler result files `*.ocsf.json`
For Kubernetes Scanning, there is the section `resources` for every json entity. One Attribute is `labels` that is always empty.
I would've expected that the labels from Kubernetes Ressources would be there.

Use Case:
We are postprocessing the files.
If we e.g. label a deployment with a responsible and another workload with a different one, we would like to notify them. There are also other use cases why this info could be useful. 

Therefor my question -> should the resource labels be in the result files or does the result file entity have a different background?

**Comment by jfagoagas** [comment]
Hello @mirisbowring we are going to review it and we'll get back to you soon.

Thanks!

**Comment by MrCloudSec** [comment]
Hi @mirisbowring, at the moment, Prowler does retrieve the tags on Kubernetes resources. Can you open a feature request so we can include it in our roadmap? Thanks.
